src/index.ts

```diff
@@ -14,35 +15,35 @@ async function getState(): Promise<any> {
         method: 'GET',
         baseURL: process.env.SALTYBET_URL,
         url: `/state.json?t=${(new Date()).getTime()}`
-    })
+    });
 
-    return response.data
+    return response.data;
 }
 
-
-let exit = false
+let exit = false;
 dotenv.config();
-const client = new Discord.Client()
+const client = new Discord.Client();
 client.on('ready', async () => {
-    console.log(`Logged in as ${client?.user?.tag}`)
+    const state = await getState();
+    let saltyBet = new SaltyBet(state.status, state.p1total, state.p2total);
```

Comment:
can we rename this to something that expresses that this is the _previous_ state? Took me a sec to understand the updates on #L41